logset.go

```diff
@@ -61,20 +70,23 @@ type LogSetReadRequest struct {
 }
 
 type LogSetReadResponse struct {
-	LogSet LogSet
+	LogSet
 	ApiResponse
 }
 
 func (l *LogSetClient) Read(readRequest LogSetReadRequest) (*LogSet, error) {
-	userClient := NewUserClient(l.AccountKey)
-	response, err := userClient.Read(UserReadRequest{})
+	resp, err := l.Get("/hosts/" + readRequest.Key)
```

Comment:
Sounds like we should keep the old model then (POST to get_user with load_hosts=1) then dig into the response json.